source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -62,12 +62,16 @@ Episode 129 of the EmberWeekend podcast featured [@mansona](https://github.com/m
 
 [@mansona](https://github.com/mansona) also introduced Guidemaker:  A fully functional, static site implementation of a documentation site and EmberJS with fully out of the box SEO friendly output! With support to be hosted on S3 or any other site hosting solution. Check out the [github](https://github.com/empress/guidemaker) repo here.
 
-Listen to the full [podcast]( https://emberweekend.com/episodes/empress-the-ember-press). 
+Listen to the full [podcast]( https://emberweekend.com/episodes/empress-the-ember-press).
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Singleton Record Data RFC](https://github.com/emberjs/rfcs/pull/461) 💾
+Want to learn more about how some of Ember Data's API's are developing? Take a peek at the Singleton Record Data RFC recently opened by [@runspired](https://github.com/runspired).
+
+This RFC focuses on ensuring that Record Data can be implemented as a singleton, eliminates some redundant API's and simplifies method signatures. This plan for Record Data offers opportunities for performance optimizations ⚡️ and improved feature sets! ✨
```

Comment:
I'd also go for `APIs` without the apostrophe here since the word doesn't [show any possession](https://en.oxforddictionaries.com/punctuation/apostrophe), but instead is only used to indicate the plural form. Do you think this works as well?

```suggestion
This RFC focuses on ensuring that Record Data can be implemented as a singleton, eliminates some redundant APIs and simplifies method signatures. This plan for Record Data offers opportunities for performance optimizations ⚡️ and improved feature sets! ✨
```

Reply:
Yep, same here!